parts/k8s/kubernetescustomscript.sh

```diff
@@ -1,19 +1,24 @@
 #!/bin/bash
-
-# This script runs on every Kubernetes VM
-# Exit codes represent the following:
-# | exit code number | meaning |
-# | 20 | Timeout waiting for docker install to finish |
```

Comment:
Keeping these "meanings" in long-form, next to the error codes variables would be **AWESOME**!